### PR TITLE
Create & Get blob with composite id

### PR DESCRIPTION
**Pull request by anandjakhaniya**

### CODE REVIEW

File: backend/src/main/java/ai/verta/modeldb/versioning/blob/factory/DatasetBlobFactory.java [hunk 1]
```diff
@@ -100,11 +100,11 @@ static PathDatasetBlob getPathBlob(Session session, String blobHash) {
...
   private List<KeyValue> getAttributes(Session session) throws InvalidProtocolBufferException {
     String getAttributesHQL =
-        "From AttributeEntity kv where kv.blob_hash = :blobHash "
-            + " AND kv.entity_name = :entityName AND kv.field_type = :fieldType AND kv.blob_hash = :blobHash";
+        "From AttributeEntity kv where kv.entity_hash = :entityHash "
+            + " AND kv.entity_name = :entityName AND kv.field_type = :fieldType";
     Query getQuery = session.createQuery(getAttributesHQL);
     getQuery.setParameter("entityName", ModelDBConstants.BLOB);
-    getQuery.setParameter("blobHash", getElementSha());
+    getQuery.setParameter("entityHash", getElementSha());
```
Comment (anandjakhaniya) [inline backend/src/main/java/ai/verta/modeldb/versioning/blob/factory/DatasetBlobFactory.java]: I have facing an issue here for creating a composite key to fetch attributes. So here I do not have repoId and commit_hash so need to figure it out how to pass it to getBlob() hierarchy. cc @ravishetye

**Comment by ravishetye** [comment]
this will need some modification on us getting the compositeID string

**Review by ravishetye — APPROVED** [review]
this might need somefollow up work , but looks good to be merged into feature branch for now

**Comment by anandjakhaniya** [comment]
Follow up work for [CompositeId](https://github.com/VertaAI/modeldb/pull/756#issuecomment-643064761) is done on PR: https://github.com/VertaAI/modeldb/pull/774

**Pull request merged**